doc/kafo_run.uml

```diff
@@ -105,5 +105,7 @@
 
   #Greenyellow:**post**|
 
+  #Greenyellow:**pre_exit**|
```

Comment:
Should every hook have a line with `exit()` to `pre_exit`?

An example: let's say a `pre_commit` hook calls `exit()`, is `pre_exit` still called? (the code suggests it does, but the chart should document this I think).

Reply:
Anything that exits, or errors can lead to this occurring and calling the exit handler hook which would make for quite a messy diagram.

I can not seem to find what syntax this is using as it's quite old and plantuml has gone through changes. I also experimented with various attempts at this and none of them made it clearer.